cookbook/import_export/automate-import-exports.rst

```diff
@@ -8,14 +8,14 @@ The batch job command
 
 Akeneo PIM provides a simple command to launch jobs:
 
-.. code-block:: batch
+.. code-block:: bash
     :linenos:
 
     app/console akeneo:batch:job [-c|--config CONFIG] [--email EMAIL] [--no-log] [--] <code>
```

Comment:
There is a note just above
"Don't forget to add --env=prod to avoid memory leaks in dev environement (the default environment for commands)"